benchmark/benchmark.c

```diff
@@ -16,6 +16,7 @@
 #include <serial_config.h>
 #include <ethernet_config.h>
 
+#ifdef CONFIG_ARCH_ARM
```

Comment:
`#ifdef`s in code files need to be factored out.  In this case, make separate benchmark-arm.c and benchmark-risc-V.c and either build just one of them in the Makefile, or include just one with
```

#include "benchmark-" #ARCH ".c"
```
or similar.

Reply:
The problem is that because there's no seL4 support for benchmarking functionality on RISC-V, we can't really do anything on RISC-V.